tests/cli/test_worker.py

```diff
@@ -503,10 +503,9 @@ async def test_install_policy_prompt(self, kubernetes_work_pool, monkeypatch):
             ],
             user_input=readchar.key.ENTER,
             expected_output_contains=[
-                (
-                    "Could not find a kubernetes worker in the current"
-                    " environment. Install it now?"
-                ),
+                "Could not find the Prefect integration library for the",
+                "kubernetes",
+                "Install the library now?",
```

Comment:
The original code here I think is what we want. your change is now checking for 3 distinct substrings of the output, which seems unnecessary

Reply:
Lol on n f string. The wrapping keeps getting screwed up, even copy pasting. Talked to Alex and decided to just make it separate list of strings.

Reply:
you could always just omit the commas

```python
In [1]: [
   ...:     "foo "
   ...:     "bar "
   ...:     "baz ",
   ...:     42
   ...: ]
Out[1]: ['foo bar baz ', 42]
```